docs/reference/config.md

```diff
@@ -1144,8 +1144,22 @@ The `manifest` scope allows you to define some meta-data information needed when
 The following settings are available:
 
 `manifest.author`
+: :::{deprecated} 24.09.0-edge
+  Use `manifest.contributors` instead.
+  :::
 : Project author name (use a comma to separate multiple names).
 
+`manifest.contributors`
+: :::{versionadded} 24.09.0-edge
+  :::
+: List of project contributors. Should be a list of maps. The following fields are supported in the contributor map:
+  - `name`: the contributor's name 
+  - `affiliation`: the contributor's affiliated organization
+  - `email`: the contributor's email address
+  - `github`: the contributor's GitHub URL
```

Comment:
We all love github, but it's not too specific, what if I'm using some other code management or social ?

Reply:
These are just the fields from the BCO spec

Reply:
fair enough

Reply:
Maybe for the description out the hubs in brackets?